2020-02-13 09:22:17,402 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:22:34,151 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:27:36,650 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:27:52,876 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:33:33,449 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:34:39,901 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:35:11,039 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:36:10,961 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 09:58:12,870 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:00:44,750 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:08:17,445 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:10:25,737 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:18:18,122 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:18:22,485 ERROR: Exception on /index [GET] [in c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\routes.py", line 21, in index
    return render_template('index.html', title='HomePage', tickets=tickets)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\templates\base.html", line 32, in top-level template code
    <a class="nav-link" href="{{ url_for('settings') }}">Options</a>
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\flask\helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "c:\users\josep\onedrive\documents\programming\portfolio\bugtracker\env\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'settings'. Did you mean 'static' instead?
2020-02-13 10:23:26,928 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:24:50,084 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:30:25,753 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:33:12,611 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:33:34,169 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:37:42,522 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:38:16,016 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:39:28,950 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:39:52,451 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:41:49,403 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:44:09,938 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 10:49:11,949 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 12:27:55,836 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 12:29:25,207 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
2020-02-13 15:20:20,685 INFO: bugTracker startup [in C:\Users\josep\OneDrive\Documents\Programming\Portfolio\bugTracker\app\__init__.py:43]
